libs/langchain/langchain/document_loaders/browserbase.py

```diff
@@ -0,0 +1,3 @@
+from langchain_community.document_loaders.browserbase import BrowserbaseLoader
```

Comment:
Could you remove this -- we're about to break dependency of langchain on langchain_community